tensorflow/python/ops/math_grad.py

```diff
@@ -19,6 +19,7 @@
 from tensorflow.python.eager import context
 from tensorflow.python.framework import constant_op
 from tensorflow.python.framework import dtypes
+from tensorflow.python.framework import indexed_slices as indexed_slices_lib
```

Comment:
We need to add this dependency to the BUILD file for math_grad

Reply:
Done